Subject: use system library instead of embeded version
From: Aron Xu <aron@debian.org>
Index: opencc/src/CMakeLists.txt
===================================================================
--- opencc.orig/src/CMakeLists.txt
+++ opencc/src/CMakeLists.txt
@@ -43,9 +43,8 @@ set(
   UTF8Util.cpp
 )
 
-include_directories(../deps/darts-clone)
-include_directories(../deps/rapidjson-0.11)
-include_directories(../deps/tclap-1.2.1)
+include_directories(/usr/include/rapidjson)
+include_directories(/usr/include/tclap)
 
 add_library(
   libopencc
